resources/client/animations/animation.service.ts

```diff
@@ -0,0 +1,194 @@
+import { newPhoneProp, removePhoneProp } from '../functions';
+import { Delay } from '../../utils/fivem';
+
+export enum AnimationState {
+  ON_CALL,
+  PHONE_OPEN,
+  ON_CAMERA,
+}
+
+export class AnimationService {
+  private animationInterval: NodeJS.Timeout;
+  private onCall: boolean = false;
+  private phoneOpen: boolean = false;
+  private onCamera: boolean = false;
+
+  createAnimationInterval() {
+    this.animationInterval = setInterval(async () => {
+      const playerPed = PlayerPedId();
+      if (this.onCall) {
+        this.handleCallAnimation(playerPed);
+      } else if (this.phoneOpen && !this.onCamera) {
+        this.handleOpenAnimation(playerPed);
+      }
+    }, 250);
+  }
+
+  setPhoneState(state: AnimationState, stateValue: boolean) {
```

Comment:
Functions like these can be set to private as they should only be accessible within the class.

Reply:
The vast majority of this can be changed to be private, the only things that should be exposed is open/closePhone, and start/endPhoneCall